### PR TITLE
TF-210: Send hand presence events even before hands have been found

### PR DESCRIPTION
**Pull request by NatGreenwood-UL**
- [x] Test plan updated (by PR creator) 
- [x] Code Review 
- [x] Developer Testing: Key item tests 
- [x] Developer Testing: Execution of the updated test plan (changed items for this work only) 
- [ ] Product Owner Review (Optional) 
- [x] Update Release Notes on the JIRA ticket for harvest at release time

### CODE REVIEW

**Comment by NatGreenwood-UL** [comment]
I think that test 452 covers the use case for this test already, and this just covers an odd case of it?

**Review by MattGrayUL — DISMISSED** [review]

**Comment by NatGreenwood-UL** [comment]
Marking as a draft PR as it has the contents of the release branch in it, can be merged once the Release is approved & the release branch merged into `develop`.

File: .gitlab-ci.yml
Comment (PeterNancollis-UH) [inline .gitlab-ci.yml:13]: Should this be 2.0.0 as it's merging to develop before the release branch gets merged?

**Review by PeterNancollis-UH — APPROVED** [review]

**Pull request merged**
